### PR TITLE
Test acs

### PR DESCRIPTION
**Pull request by avdata99**
Add test for login responses

### CODE REVIEW

**Comment by mbocevski** [comment]
@avdata99 looks like a good approach. Is this PR still work in progress to add unsigned and signed responses?

**Comment by avdata99** [comment]
Thanks, @mbocevski. Yes, it's in progress, I just move to draft

File: bin/travis-build.bash [hunk 1]
```diff
@@ -5,6 +5,7 @@ echo "This is travis-build.bash..."
 
 echo "Installing the packages that CKAN requires..."
 sudo apt-get update -qq
+sudo apt-get install xmlsec1
```
Comment (mbocevski) [inline bin/travis-build.bash:8]: @avdata99 you need to also install `libxmlsec1-dev`, so that pip can build xmlsec.

**Comment by avdata99** [comment]
This is finally ready for review @adborden @mbocevski 
We now are testing encrypted assertions.

**Comment by mbocevski** [comment]
Great stuff @avdata99! This would be great to port to `main` as well. Let's go through the review first. Do we have to run tests with debug on? Is that due to that we want to get the full SAML responses?

**Comment by avdata99** [comment]
My plan is to port this to `main` if the review it's ok @mbocevski.
We don't really need `debug` but it was helpful to understand all this process. We can remove it.

**Comment by mbocevski** [comment]
>We don't really need debug but it was helpful to understand all this process

I definitely get that! I suggest you clean it up so that we don't have debug output in CI and you can put comments for other devs to know how they can troubleshoot and debug the flows.

**Comment by duskobogdanovski** [comment]
I would suggest to move the tests in [test_blueprint.py](https://github.com/keitaroinc/ckanext-saml2auth/blob/ckan-2.8/ckanext/saml2auth/tests/test_blueprint.py) or rename the test module with more intuitive name. (e.g. test_blueprint_get_request.py)

**Comment by avdata99** [comment]
> I would suggest to move the tests in [test_blueprint.py](https://github.com/keitaroinc/ckanext-saml2auth/blob/ckan-2.8/ckanext/saml2auth/tests/test_blueprint.py) or rename the test module with more intuitive name. (e.g. test_blueprint_get_request.py)

@duskobogdanovski I just rename the test file. rebase to `ckan-2.8` and squash all commits

**Pull request merged**